lib/std/builtin.zig

```diff
@@ -21,10 +21,10 @@ pub const SubSystem = std.Target.SubSystem;
 /// Deprecated: use `std.Target.Cpu`.
 pub const Cpu = std.Target.Cpu;
 
-/// `explicit_subsystem` is missing when the subsystem is automatically detected,
-/// so Zig standard library has the subsystem detection logic here. This should generally be
-/// used rather than `explicit_subsystem`.
-/// On non-Windows targets, this is `null`.
```

Comment:
This line break is intentional

Reply:
Whoops, removing it wasn't; that was an artifact of my line wrapper.